encoding/byteString/index.js

```diff
@@ -0,0 +1,6 @@
+import _from from "./from.js";
```

Comment:
why underscore?

Reply:
`I was pretty sure `from` was a reserved keyword just like `function`.  I am gonna investigate that.

Reply:
Yes, you were right. `from` is not a reserved keyword and does not need aliasing.